shared/src/main/java/uk/gov/di/authentication/shared/dynamodb/DynamoDBSchemaHelper.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.di.authentication.shared.dynamodb;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapperConfig;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.amazonaws.services.dynamodbv2.model.Delete;
+
+import java.util.Map;
+
+public class DynamoDBSchemaHelper {
+
+    private final AmazonDynamoDB dynamoDB;
+    private final String environment;
+
+    public DynamoDBSchemaHelper(AmazonDynamoDB dynamoDB, String environment) {
+        this.dynamoDB = dynamoDB;
+        this.environment = environment;
+    }
+
+    public enum Table {
+        USER_CREDENTIALS_TABLE("user-credentials", "Email"),
+        USER_PROFILE_TABLE("user-profile", "Email");
+
+        private String tableName;
+        private String key;
```

Comment:
These can both be `final`

Reply:
We could be explicit with the key as `primaryKey`

Reply:
updated